src/main/java/frc/trigon/robot/poseestimation/poseestimator/PoseEstimatorConstants.java

```diff
@@ -15,7 +15,7 @@ public class PoseEstimatorConstants {
      */
     static final Vector<N3> ODOMETRY_AMBIGUITY = VecBuilder.fill(0.003, 0.003, 0.0002);
 
-    static final double
+    public static final double
             TRANSLATIONS_STD_EXPONENT = 0.005,
             THETA_STD_EXPONENT = 0.01;
```

Comment:
delete this, and move to camera constants to be calibrated 
for us, it'll be the same for all the cameras, but once you have a different one we'll have multiple variables